odoo/fields.py

```diff
@@ -4869,12 +4877,13 @@ def relation_delete(ys):
         pairs = [(x, y) for x, ys in new_relation.items() for y in ys - old_relation[x]]
         if pairs:
             if self.store:
-                query = SQL("INSERT INTO {} ({}, {}) VALUES %s ON CONFLICT DO NOTHING").format(
-                    Identifier(self.relation),
-                    Identifier(self.column1),
-                    Identifier(self.column2),
-                )
-                cr.execute_values(query, pairs)
```

Comment:
Note that we don't use the default `page_size` anymore, which could have an impact on performances

Reply:
I have never seen a performance report for that one.  Its conversion from `execute()` to `execute_values()` was a side effect from another task, and was not justified.